### PR TITLE
add license infos and more

### PR DESCRIPTION
**Pull request by gzussa**

### CODE REVIEW

File: LICENSE-3rdparty.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Component,Origin,License,Copyright
+import,decorator,BSD 2-Clause,Copyright (c) 2005-2018, Michele Simionato
```
Comment (jeremy-lq) [inline LICENSE-3rdparty.csv]: Please use SPDX identifiers for the license column.

**Comment by gzussa** [comment]
The stale Github actions needs to be fixed before we can merge this PR

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -107,3 +84,82 @@ Now you can use your new SDK and call the following methods with various params
...
+- [DogStatsD](/datadog/dogstatsd)
+- [Threadstats](/datadog/threadstats)
+
+
+We have [unit](/tests/unit), [integration](/tests/integration) and [performance](/tests/performamce) tests.
+Integration tests need an _API_ and _APP Keys_ to run against a Datadog account.
+- __WARNING__: Never use keys for an organization that contains anything important.
+
+We use `tox` to run tests. You can find the [tox.ini](/tox.ini) config in the root directory.
+We create 2 environments:
```
Comment (zippolyte) [inline DEVELOPMENT.md]: technically, there are 10 environments:  
- the 9 that are run when invoking simply `tox`. Every pyXX env runs unit tests:
```
❯ tox -l
py27
py35
py36
py37
py38
pypy2
pypy3
flake8
integration
```
- the `integration-admin` environment that you have to explicitly call: `tox -e integration-admin`

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -107,3 +84,82 @@ Now you can use your new SDK and call the following methods with various params
...
+- The Default environment: It will run all Unit tests, Performance test and Integration tests not marked as `admin_needed`.
+  - Execute this with the `tox` command.
+- The `integration-admin` environment: It will only run integration test marked with the `admin_needed` marker.
```
Comment (zippolyte) [inline DEVELOPMENT.md]: integration tests

File: DEVELOPMENT.md [hunk 4]
```diff
@@ -107,3 +84,82 @@ Now you can use your new SDK and call the following methods with various params
...
+You can also install this project locally in editable mode to make changes and run any manual tests. 
+This can be done by installing using the following pip command:
+
+```
+pip install -e <path_to_cloned_project_dir>
+```
+
+Push to your fork and [submit a pull request][pr].
+
+[pr]: https://github.com/your-username/datadogpy/compare/DataDog:master...master
```
Comment (zippolyte) [inline DEVELOPMENT.md]: let's not link since it's not a real one

File: LICENSE [hunk 5]
```diff
@@ -1,24 +1,26 @@
-Copyright (c) 2015, Datadog <info@datadoghq.com>
-All rights reserved.
+Copyright (c) 2010-2020, Datadog <opensource@datadoghq.com>
```
Comment (nmuesch) [inline LICENSE]:
```suggestion
Copyright (c) 2015-Present, Datadog <opensource@datadoghq.com>
```

Comment (nmuesch) [reply LICENSE]: We can just do that right ^ ?

File: datadog/api/constants.py [hunk 6]
```diff
@@ -1,3 +1,8 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the BSD-3-Clause License.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2020 Datadog, Inc.
+
+
```
Comment (nmuesch) [inline datadog/api/constants.py]: double newline

File: datadog/threadstats/constants.py [hunk 7]
```diff
@@ -1,3 +1,8 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the BSD-3-Clause License.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2020 Datadog, Inc.
+
+
```
Comment (nmuesch) [inline datadog/threadstats/constants.py]: do we need double newlines?

**Comment by nmuesch** [comment]
We should be all set to add back the stale yml file now too 😄

File: RELEASING.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+This document summarizes the process of doing a new release of this project.
+Release can only be performed by Datadog maintainers of this repository.
+
+## Schedule
+This project does not have a strict release schedule. However, we would make a release at least every 2 months.
+  - No release will be done if no changes got merged to the `master` branch during the above mentioned window.
+  - Releases may be done more frequently than the above mentioned window.
+  - Create a pager duty schedule corresponding to this.
+  - Create a google calendar schedule corresponding to this.
```
Comment (nmuesch) [inline RELEASING.md]: Should we keep these two lines here?

Comment (gzussa) [reply RELEASING.md]: definitely not 🤦‍♂ !

File: .github/labeler.yml [hunk 9]
```diff
@@ -0,0 +1,3 @@
+# Configuration for labeler - https://github.com/actions/labeler
+
+documentation: "*.md"
```
[the remark this answers is not in the document]
Comment (zippolyte) [reply .github/labeler.yml]:
```suggestion
documentation: "**/*.md"
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -1,107 +1,111 @@
...
+| `duplicate`                   | Issue Only               | Duplicate issue.
+| `invalid`                     | Issue Only               | Invalid issue.
+| `kind/bug`                    | Issue Only               | Bug related issue.
+| `kind/feature-request`        | Issue Only               | Feature request related issue.
+| `severity/critical`           | Issue Only               | Critical severity issue.
+| `severity/major`              | Issue Only               | Major severity issue.
+| `severity/minor`              | Issue Only               | Minor severity issue.
+| `severity/normal`             | Issue Only               | Normal severity issue.
+| `stale`                       | Issues and Pull Requests | Stale - Bot reminder.
+
```
[the remark this answers is not in the document]
Comment (zippolyte) [reply CONTRIBUTING.md]:
```suggestion
| `stale/exempt`                | Issues and Pull Requests | Exempt from being marked as stale.
```

File: datadog/api/constants.py (hunk 6 above)
Comment (zippolyte) [reply datadog/api/constants.py]:
```suggestion
```

File: datadog/threadstats/constants.py (hunk 7 above)
Comment (zippolyte) [reply datadog/threadstats/constants.py]:
```suggestion
```

**Pull request merged**